### PR TITLE
[release-1.13] Run update_deps.sh

### PR DESCRIPTION
**Pull request by ericvn**
**Please provide a description of this PR:**

Run update_deps.sh for 1.13. The proxy build isn't done so the test may fail until that proxy artifact is available.

### CODE REVIEW

**Review by lei-tang — APPROVED** [review]

File: docker/Dockerfile.distroless [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # prepare a distroless source context to copy files from
-FROM gcr.io/distroless/static-debian11@sha256:21d3f84a4f37c36199fd07ad5544dcafecc17776e3f3628baf9a57c8c0181b3f as distroless_source
+FROM gcr.io/distroless/static-debian11@sha256: as distroless_source
```
Comment (stewartbutler) [inline docker/Dockerfile.distroless:2]: This doesn't look right.

Comment (ericvn) [reply docker/Dockerfile.distroless:2]: Not sure why these are wrong, but running commands themselves so no changes required.

**Pull request merged**